src/Security/Authentication/JwtBearer/src/JwtBearerOptions.cs

```diff
@@ -15,13 +16,17 @@ namespace Microsoft.AspNetCore.Authentication.JwtBearer;
 public class JwtBearerOptions : AuthenticationSchemeOptions
 {
     private readonly JwtSecurityTokenHandler _defaultHandler = new JwtSecurityTokenHandler();
+    private readonly JsonWebTokenHandler _defaultTokenHandler = new JsonWebTokenHandler();
```

Comment:
```suggestion
    private readonly JsonWebTokenHandler _defaultTokenHandler = new JsonWebTokenHandler
    {
        MapInboundClaims = JwtSecurityTokenHandler.DefaultMapInboundClaims,
    }
```

Right? Otherwise it doesn't align with the default value of `_mapInboundClaims`. Can we add a test to [JwtBearerTests](https://github.com/dotnet/aspnetcore/blob/da8a6f0766d1f333cc2faec72f97751e3c605dac/src/Security/Authentication/test/JwtBearerTests.cs) verifying that we default to mapping claims when `UseTokenHandlers` is configured?

Reply:
@halter73 thanks for pointing this out, i will have a look

Reply:
@halter73 About this comment:
>How breaking would it be to default to using the new TokenHandlers? We could in theory make it implicitly opt-in only if you touch the legacy SecurityTokenValidators properties. This could help us avoid adding a new UseTokenHandlers bool to the options.

For many users, they will never need to touch SecurityTokenValidators.

Reply:
@kevinchalet I am thinking that

Reply:
Addressed: 507d24137b37c43ddf9c18bdeb7510e70c84bcf1